server/service/src/processors/transfer/requisition/test.rs

```diff
@@ -90,41 +90,54 @@ async fn requisition_transfer() {
     let test_handle = exec_concurrent(
         test_input,
         number_of_instances,
-        |_, test_input| async move {
+        |thread_num, test_input| async move {
```

Comment:
Hey James, I'm struggling to figure out how this closure is actually finding the thread count. Would you be willing to explain how it is capturing it?

Reply:
@fergie-nz have a look at the exec_concurrent code, it's generated in there...
It's just a for loop based on the number of concurrent instances